libs/ledger-live-common/src/families/cosmos/js-signOperation.ts

```diff
@@ -50,14 +50,35 @@ const signOperation: SignOperationFnSignature<Transaction> = ({ account, deviceI
             accountNumber.toString(),
             sequence.toString(),
           );
-          const ledgerSigner = new LedgerSigner(transport, {
-            hdPaths: [stringToPath("m/" + account.freshAddressPath)],
-            prefix: cryptoFactory(account.currency.id).prefix,
+          const tx = Buffer.from(JSON.stringify(sortObjectKeysDeeply(signDoc)), "utf-8");
```

Comment:
we can replace the sortObjectKeysDeeply with serializeSignDoc as I mentioned above

Reply:
Done